packages/playground-preview-worker/src/index.ts

```diff
@@ -367,12 +367,15 @@ app.onError((e, c) => {
 	const sentry = c.get("sentry");
 	const registry = c.get("prometheus");
 
-	const errorCounter = registry.create(
-		"counter",
-		"devprod_playground_preview_worker_error_total",
-		"Error counter for DevProd's playground-preview-worker service"
-	);
-	errorCounter.inc();
+	// Only include reportable `HttpError`s or any other error in error metrics
+	if (!(e instanceof HttpError) || e.reportable) {
+		const errorCounter = registry.create(
+			"counter",
+			"devprod_playground_preview_worker_error_total",
+			"Error counter for DevProd's playground-preview-worker service"
+		);
+		errorCounter.inc();
```

Comment:
Would there be value in including the error message here?
```suggestion
		errorCounter.inc({ message: e.message });
```

Reply:
I think I read somewhere that it was bad practice to include error messages in Prometheus counts because it caused creation of new "collections(?)" for each error phrase.

Reply:
Oh, yeah. What @mrbbot said above. (Sorry I missed that comment).

Reply:
Makes sense - thought it would be worth checking anyway!